api/config.go

```diff
@@ -45,7 +45,11 @@ type Config struct {
 	// It changes every time the configuration is changed and cannot be updated.
 	UUID string `json:"uuid,omitempty"`
 	// Routable determines if the application should be exposed by the router.
-	Routable bool `json:"routable,omitempty"`
+	Routable bool `json:"routable"`
+}
+
+func NewConfig() Config {
+	return Config{Routable: true}
```

Comment:
That is true. I guess that means we have to go the original route with using a bool pointer instead.